File: BE-System-checkBV01/src/main/java/com/checkxmlbv01/websitecheckxmlbv01/service/BacSi/BacSiService.java
package com.checkxmlbv01.websitecheckxmlbv01.service.BacSi;

import java.util.List;

import com.checkxmlbv01.websitecheckxmlbv01.domain.BacSi;

public interface BacSiService {

    BacSi create(BacSi bacSi);

    BacSi update(Long id, BacSi bacSi);

    void delete(Long id);

    BacSi getById(Long id);

    List<BacSi> getAll();
}

File: FE-System-checkBV01/src/services/bacSiService.js
import axiosClient from "./axiosClient";

const API = "/api/v1/bac-si";

export const getAll = () => axiosClient.get(API);
export const getById = (id) => axiosClient.get(`${API}/${id}`);
export const create = (data) => axiosClient.post(API, data);
export const update = (id, data) => axiosClient.put(`${API}/${id}`, data);
export const remove = (id) => axiosClient.delete(`${API}/${id}`);
